nav2_bringup/bringup/params/nav2_params.yaml

```diff
@@ -51,6 +51,7 @@ bt_navigator:
     use_sim_time: True
     global_frame: map
     robot_base_frame: base_link
+    odom_topic: /odom
```

Comment:
Is that leading slash actually required?

Reply:
I don't think so but that's how the other topics were set in the params file, added it to maintain the symmetry

Reply:
OK sounds good